hub/core/query/query.py

```diff
@@ -198,7 +198,11 @@ def size(self):
         return self.numpy().size  # type: ignore
 
     def contains(self, o: object) -> bool:
-        return any(self.numpy() == o)  # type: ignore
+        v = self.numpy() == o
```

Comment:
self.numpy() might resolve to a scalar, Those comparison would be performed on `scalar == scalar`, instead of `ndarray == scalar`. I have a feeling that this logic is indeed a bit broken somewhere. But really didn't have a chance to improve on it.

Reply:
`np.any` returns a scalar bool for both cases: when `self.numpy()` is a scalar or is an ndarray, which is what the if block is doing, right?  So the if-else block can be replaced by `np.any(self.numpy() == o)`.